tests/fixtures/metrics_fixtures.py

```diff
@@ -59,11 +79,33 @@ def test_implicit_labels():
     ], dtype=float)
 
 
+@pytest.fixture()
+def test_implicit_labels_negative_sample_user():
+    return torch.tensor([
+        [1, 0, 1],
+        [1, 1, 0],
+        [1, 1, 1],
+        [0, 0, 1],
+    ], dtype=float)
+
+
 @pytest.fixture()
 def targets(test_implicit_interactions):
     return test_implicit_interactions.mat.tocsr()
 
 
+@pytest.fixture()
+def targets_auc_negative_sample_user():
+    # ``sklearn.metrics.roc_auc_score`` throws an error if an item has interacted
+    # with every user so changing the final item
+    return Interactions(
+        users=[0, 0, 0, 1, 1, 1, 2, 2],
+        items=[0, 1, 2, 1, 2, 3, 0, 3],  # changed final item
+        ratings=[1, 1, 1, 1, 1, 1, 1, 1],
+        check_num_negative_samples_is_valid=False,
+    ).mat.tocsr()
```

Comment:
Why make this a separate fixture and not just update the original `targets` fixture?